docs/run/native-gates.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Native gates and operations
+description: Summary of the native gates and operations supported by IBM Quantum systems
+
+---
+
+# Native gates and operations
+
+Each [processor family](processor-types) has a native gate set, and the systems in each family support running only those particular gates and operations. Transpilation translates an arbitrary circuit to the native gate set supported by a specified system.  You can view the native gates and operations for a system either [with Qiskit](#native-gates-with-qiskit) or on the IBM Quantum Platform [Compute resources page](#native-gates-on-platform).
```

Comment:
It feels weird to just have a link to Basis gates at the end when they've never been mentioned in the topic.  Maybe something like this: 

Each [processor family](processor-types) has a native gate set.  By default, the systems in each family only support running the gates and operations in the native gate set. Thus, every gate in the circuit must be translated (by the transpiler) to the elements of this set.   

You can view the native gates and operations for a system either [with Qiskit](#native-gates-with-qiskit) or on the IBM Quantum Platform [Compute resources page](#native-gates-on-platform).

Note: The terms _native gate set_ and _basis gates_ are often used interchangeably.  However, you can specify a different set of basis gates to use, while the native gate set never changes.  For information about changing the basis gates, see https://docs.quantum.ibm.com/transpile/representing_quantum_computers#basis-gates

Reply:
Very nice wording! Would you remove the Next steps section, or keep it as-is in addition to adding this note in the first section?